data/json/mutations/mutations.json

```diff
@@ -587,7 +587,7 @@
     "id": "DISRESISTANT",
     "name": { "str": "Disease Resistant" },
     "points": 1,
-    "description": "It's very unlikely that you will catch ambient diseases like a cold or the flu.",
+    "description": "It's very unlikely that you will catch ambient diseases like the cold or flu.",
```

Comment:
```suggestion
    "description": "It's very unlikely that you will catch ambient diseases like a cold or flu.",
```

Reply:
Hrm. Looking at this, I realize I might need to walk back; we tend to use the indefinite article for "a cold" and the definite article for "the flu," but stapling them back to back like this is weird. My brain wants to use the definite article in this case.

Reply:
I've never heard someone say "the cold"; I have heard someone say "a flu". But I don't feel that strongly about it.